lib/cli/scripts/scan-env.ts

```diff
@@ -69,6 +73,18 @@ async function getPeopleCount(skipCount: number = 0): Promise<PeopleTally> {
     }
 }
 
+async function getHomeFoldersCount(): Promise<number> {
+    try {
+        const nodesApi = new NodesApi(jsApiConnection);
+        const rootApiResult = await nodesApi.listNodeChildren('-root-');
```

Comment:
are we sure that there is no way to pass a relative path such as 
`listNodeChildren('-root-', {
            'relativePath': 'User Homes',
            'maxItems': 500
        });`

Reply:
In addition to that keep in mind that the api will return the matItems so you have to iterate to get all of them!

Reply:
Oh yes you're correct, I had missed that thanks. Updating now

Reply:
can we please have the User Homes as a constant

Reply:
Only saw this after committing. Updating again